internal/services/cdn/cdn_profile_resource_test.go

```diff
@@ -133,6 +133,28 @@ func TestAccCdnProfile_createShouldFail(t *testing.T) {
 	})
 }
 
+func TestAccCdnProfile_locationNormalization(t *testing.T) {
```

Comment:
Since this is for an api version we can't test, we don't need to have this test here